iis-core/src/test/java/eu/dnetlib/iis/core/OozieWorkflowTestConfiguration.java

```diff
@@ -30,6 +33,9 @@
 	private int timeoutInSeconds = defaultTimeoutInSeconds;
 	private WorkflowJob.Status expectedFinishStatus = defaultExpectedFinishStatus;
 	
+	private List<String> outputFilesToInclude = Lists.newArrayList();
```

Comment:
The comment about the timeout starting with 'I had to change...' is not general enough. I didn't spot it earlier.